cluster/gce/config-test.sh

```diff
@@ -246,8 +246,22 @@ if [ ${ENABLE_IP_ALIASES} = true ]; then
   PROVIDER_VARS="${PROVIDER_VARS} ENABLE_IP_ALIASES"
 fi
 
+# Admission Controllers to invoke prior to persisting objects in cluster
+# If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
+ADMISSION_CONTROL="${KUBE_ADMISSION_CONTROL:-NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,PodPreset,DefaultStorageClass,DefaultTolerationSeconds}"
+
+# Optional: use a default, fairly open PodSecurityPolicy
+ENABLE_POD_SECURITY_POLICY="${ENABLE_POD_SECURITY_POLICY:-true}"
+
+if [[ "${ENABLE_POD_SECURITY_POLICY:-}" == "true" ]]; then
+  ADMISSION_CONTROL="${ADMISSION_CONTROL},PodSecurityPolicy"
```

Comment:
> I think if you specify the admission chain, it should be the entire chain.

In practice, that means that many installations will have misordered chains, or miss picking up new strongly recommended plugins just because they wanted one non-default plugin in a previous release.